### PR TITLE
Add basic/syntax-errors translation

### PR DESCRIPTION
**Pull request by aliaksandr-s**

### CODE REVIEW

**Comment by aliaksandr-s** [comment]
Hey guys) Here is another chapter, @freetonik please check.

File: modules/10-basics/50-syntax-errors/description.en.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+
+name: Syntax errors
+theory: |
+  If a PHP program isn't syntactically correct, interpreter will show a relevant message and note with a file and line where an error might occur. **Syntax errors** occur when the code has grammar mistakes. Unlike human languages where grammar rules are important and the lack of it won't affect your ability to read or understand any text; everything in programming is strict. Even a tiny violation and your program will not run. Some examples could be, a wrong parenthesis order, a `;` that you forgot to add or other details.
```
Comment (freetonik) [inline modules/10-basics/50-syntax-errors/description.en.yml]: - "a message [...] **a** note" (without an article note seems like a verb here)
- "an error might occur" — wrong tense, the error already occurred. "Might have occurred" would suit better.
- "and the lack of" — should be "but", not "and"
- "...text; everything..." — I don't think the semicolon is needed here, it splits two parts of the same sentence, which is confusing (Unlike..., bla)
- "Some examples could be" — in cases like this feel free to rephrase the sentence to make it more natural in the English context. No need to preserve the Russian constructs. Also, ditch the comma :)

File: modules/10-basics/50-syntax-errors/description.en.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  Here is an example of code with a syntax error:
+
+  ```php
+  <?php
+
+  print_r('Hodor')
+  ```
+
+  If we run this code we will see the following message: `$ PHP Parse error:  syntax error, unexpected end of file in /private/var/tmp/index.php on line 4`. In PHP such errors are called "Parse error". As we can see it includes a file path and line number.
```
Comment (freetonik) [inline modules/10-basics/50-syntax-errors/description.en.yml]: - "a line number"

File: modules/10-basics/50-syntax-errors/description.en.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  On one hand, parse errors are simplest because they are all about code grammar rules and aren't related to any code logic. They can be easily fixed, you just need to find them.
```
Comment (freetonik) [inline modules/10-basics/50-syntax-errors/description.en.yml]: - "the simplest"

File: modules/10-basics/50-syntax-errors/description.en.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+  On the other hand, interpreter will not always tell you the right place of an error. Sometimes you need to add a forgotten bracket to another place as opposed to what the error message says.
```
Comment (freetonik) [inline modules/10-basics/50-syntax-errors/description.en.yml]: I think it'd be better to say "the correct position of an error".

File: modules/10-basics/50-syntax-errors/description.en.yml (hunk 1 above)
Comment (aliaksandr-s) [reply modules/10-basics/50-syntax-errors/description.en.yml]: * "where an error might occur" - probably it's better to remove this 'might' word and just say "where an error occured" 
* "and the lack of" - this sentence sounds weird with both "but" and "and", I'll just rephrase it completely.
* "Some examples could be" - yeah, I'll rephrase this one too. btw those commas were suggested by a native speaker :)

**Comment by aliaksandr-s** [comment]
@freetonik Updated

**Comment by freetonik** [comment]
Cool, thanks a lot!

**Pull request merged**